demos/prefsServerIntegration/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <link rel="stylesheet" href="../../src/lib/infusion/src/lib/normalize/css/normalize.css" />
+        <link rel="stylesheet" href="../../src/lib/foundation/css/foundation.min.css" />
+        <link rel="stylesheet" href="../../src/css/style.css" />
+        <link rel="stylesheet" href="css/style.css" />
+
+        <!-- CSS required for the Prefs Editor -->
+        <link rel="stylesheet" type="text/css" href="../../src/lib/infusion/src/framework/preferences/css/Enactors.css" />
+        <link rel="stylesheet" type="text/css" href="../../src/lib/infusion/src/framework/preferences/css/PrefsEditor.css" />
+
+        <script type="text/javascript" src="../../src/lib/infusion/infusion-custom.js"></script>
+        <script type="text/javascript" src="../../src/js/keyboardShortcut.js"></script>
+        <script type="text/javascript" src="../../src/js/ttsHookup.js"></script>
+        <script type="text/javascript" src="../../src/js/tooltip.js"></script>
+        <script type="text/javascript" src="../../src/js/msgLookup.js"></script>
+        <script type="text/javascript" src="../../src/js/selfVoicing.js"></script>
+        <script type="text/javascript" src="../../src/js/stickyKeysAssessment.js"></script>
+        <script type="text/javascript" src="../../src/js/keyboardInput.js"></script>
+        <script type="text/javascript" src="../../src/js/stickyKeysAdjuster.js"></script>
+        <script type="text/javascript" src="../../src/js/panels.js"></script>
+        <script type="text/javascript" src="../../src/js/enactors.js"></script>
+        <script type="text/javascript" src="../../src/schemas/schemas.js"></script>
+        <script type="text/javascript" src="../../src/js/navButtons.js"></script>
+        <script type="text/javascript" src="../../src/js/navIcons.js"></script>
+        <script type="text/javascript" src="../../src/js/helpButton.js"></script>
+        <script type="text/javascript" src="../../src/js/stepCount.js"></script>
+        <script type="text/javascript" src="../../src/js/nav.js"></script>
+        <script type="text/javascript" src="../../src/js/firstDiscoveryEditor.js"></script>
+        <script type="text/javascript" src="../js/demos.js"></script>
+        <script type="text/javascript" src="schemas/schemas.js"></script>
+
+        <title>First Discovery Tool</title>
```

Comment:
shouldn't this one be "First Discovery Tool - Integrated with Preferences Server"